jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -90,10 +114,20 @@ private LayerConfiguration(String label, ImmutableList<LayerEntry> layerEntries)
     this.layerEntries = layerEntries;
   }
 
+  /**
+   * Gets the label.
```

Comment:
Would `name` be better to prevent confusion with container labels?

Reply:
Yea that sounds good - or even `id`?